src/Apropos/LogicalModel.hs

```diff
@@ -32,14 +32,9 @@ enumerateScenariosWhere holds = enumerateSolutions $ logic :&&: holds :&&: allPr
     mention p = Var p :||: Not (Var p)
 
 enumerateSolutions :: LogicalModel p => Formula p -> [Set p]
-enumerateSolutions f = fromSolution <$> solveAll f
-  where
-    fromSolution :: LogicalModel p => Map.Map p Bool -> Set p
-    fromSolution m = Set.fromList $ filter isInSet enumerated
-      where
-        isInSet k = Just True == Map.lookup k m
+enumerateSolutions f = Map.keysSet . Map.filter id <$> solveAll f
```

Comment:
Why not `enumerateSolutions :: Ord p => Formula p -> [Set p]` and move  this to `Apropos.LogicalModel`?

Reply:
I.e. add it to the typeclass? I don't see why. Extra typeclass methods should be restricted to only those things that there is a good reason to override.

Reply:
Sorry. I made a typo. I meant move it to the `Apropos.LogicalModel.Formula` module.

Reply:
Agreed. It has no `LogicalModel` dependency at all. I'll see to it.

Reply:
Done and brought up to date.